client/src/pages/Home/Home.tsx

```diff
@@ -1,31 +1,116 @@
+    image: SplashImage2,
+    content: 'Not only is it delicious, but you can clean it up in a flash',
+  },
+  {
+    title: 'Resep Garang Asem Ayam Kampung',
+    image: SplashImage3,
+    content: 'Keeping It Simple · Easy Weeknight One-pot Recipes',
+  },
+];
 
 const Home = () => {
   const user: any = useSelector(selectorUser);
-  const dispatch = useDispatch();
-  const navigate = useNavigate();
 
-  const handleLogout = () => {
-    navigate('/login');
-    dispatch(logout());
-  };
+  const [currentStep, setCurrentStep] = useState<number>(0);
+  const { title, image, content } = steps[currentStep];
 
-  if (!user) return <p>loading...</p>;
+  if (!user) return <Loading />;
 
   return (
-    <div>
-      <p>Email: {user.email}</p>
-      <p>Username: {user.name}</p>
-      <p>
+    <div className='px-5 h-full select-none'>
+      <div className='flex justify-between items-center'>
         <img
           src={`data:image/${user.avatar.imageFormat};base64, ${user.avatar.base64}`}
-          alt=''
+          className='w-[45px] h-[45px] rounded-full object-cover cursor-pointer'
+          alt='default avatar'
         />
-      </p>
-      <button onClick={() => handleLogout()}>Logout</button>
+
+        <div>
+          <h1 className='text-lg font-medium leading-8'>Hi, {user.name}</h1>
+          <p className='text-sm text-gray-700'>What are you cooking today?</p>
+        </div>
+
+        <div className='flex items-center justify-center'>
+          <button type='button' className='hover:text-orange delay-100'>
+            <Icon type={IconTypes.Outlined} size={24} icon='notifications' />
+          </button>
+        </div>
+      </div>
+      <div className='relative mt-4 w-full'>
+        <h1 className='text-2xl mb-4 font-medium'>Cookbooks</h1>
+        <span className='absolute top-0 right-0 m-1'>
+          {currentStep + 1}/{steps.length}
+        </span>
+
+        <div className='border border-gray-300 rounded-2xl bg-white p-2 mt-1'>
+          <div className='relative h-2/5'>
+            <img
+              src={image}
+              className='rounded-2xl w-full h-full object-cover'
+            />
+          </div>
+
+          <div className='relative w-full h-3/5 -mt-14 flex items-center justify-center'>
+            <div className='rounded-2xl flex items-center justify-center flex-col text-center h-full mb-auto bg-white py-5 mx-4'>
+              <div className='p-3 border border-gray-400 rounded-full mb-3'>
+                <img
+                  src={UnionImage}
+                  width={20}
+                  alt='logo of mallika food recipe app'
+                />
+              </div>
+              <h1 className='text-2xl mb-3 font-medium'>{title}</h1>
+              <p className='text-sm text-gray-700 mb-3'>{content}</p>
+              <div className='relative before:absolute before:top-0 before:bottom-0 before:bg-gray-500 before:w-px flex justify-around text-center font-medium w-full items-center'>
```

Comment:
```suggestion
              <div className='relative before:absolute before:top-0 before:bottom-0 before:bg-gray-400 before:w-px flex justify-around text-center font-medium w-full items-center'>
```
Check màu trong file figma, nhấn giữ nguyên Ctrl + trỏ chuột tới vị trí element, góc phải là thấy màu

![image](https://user-images.githubusercontent.com/57981216/147955218-7f7662ee-3050-4467-9b4d-f3fc02edd6bc.png)